server/src/main/resources/org/cloudfoundry/identity/uaa/db/postgresql/V3_10_4__Add_Index_On_Identity_Zone_To_Revocable_Tokens.sql

```diff
@@ -0,0 +1,2 @@
+-- create an index to match the query
+CREATE INDEX revocable_tokens_zone_id ON revocable_tokens(LOWER(identity_zone_id));
```

Comment:
Which query are we matching? The queries accessing this are not using lower(). 

https://github.com/cloudfoundry/uaa/blob/7fffffd466ebf21cb1148f1808c92a29d0597a68/server/src/main/java/org/cloudfoundry/identity/uaa/oauth/token/JdbcRevocableTokenProvisioning.java#L41-L49